docs/additional-features/export-templates.md

```diff
@@ -18,6 +18,14 @@ Height: {{ rack.u_height }}U
 
 To access custom fields of an object within a template, use the `cf` attribute. For example, `{{ obj.cf.color }}` will return the value (if any) for a custom field named `color` on `obj`.
 
+Since local_context_data is part of an object, it is immediately available to use in the export template, however if you need the fully rendered context config you'll need to use the function `get_config_context`. For example:
```

Comment:
We should not encourage accessing `local_context_data` directly: `get_config_context()` should always be used to ensure the complete configuration data is returned.

Reply:
I've updated the comment here to drop completely the reference to `local_context_data` and said to use the config_context data you should use the `get_config_context` function.